packages/node/package.json

```diff
@@ -26,7 +26,7 @@
     "@nestjs/schedule": "^1.0.2",
     "@subql/common": "^2.2.1",
     "@subql/common-algorand": "workspace:*",
-    "@subql/node-core": "^2.4.5",
+    "@subql/node-core": "^2.5.0",
```

Comment:
I think there was a breaking change with the metadata service that was under a patch/minor release. 

Can you please update and make sure its working.